gitpod/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM gitpod/workspace-full:latest
+
+ENV DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true
+
+RUN sudo apt-get update \
+    && sudo apt-get install -y --no-install-recommends \
+        #basic tools
+        ca-certificates \
+        cmake \
+        g++ \
+        git \
+        make \
+        unzip \
+        wget \
+        # needed for ansible
+        python3-pip \
+        # needed for drawio-desktop
+        desktop-file-utils \
+        xvfb \
+    && sudo rm -rf /var/lib/apt/lists/* \
+    && sudo pip --no-cache-dir install ansible
+
+COPY --from=ghcr.io/langchr86/docmake:latest /tmp/ansible-role-docmake /tmp/ansible-role-docmake
```

Comment:
Would be more flexible to clone the origin repo (https://github.com/langchr86/ansible-role-docmake) and execute the script from there.

Reply:
I prefer to copy things from docker container, because they can be cached.
(and have defined version).